tests/serde.rs

```diff
@@ -0,0 +1,45 @@
+#![cfg(test)]
+#![cfg(all(target_family = "wasm", feature = "serde"))]
+
+use wasm_bindgen_test::wasm_bindgen_test;
+use web_time::SystemTime;
+
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+#[wasm_bindgen_test]
+/// (De)serialization of [`SystemTime`].
+async fn system_time() {
```

Comment:
```suggestion
fn system_time() {
```
None of these tests require `async`.